ModularContent/Fields/Post_List.php

```diff
@@ -535,11 +536,23 @@ protected static function get_p2p_filtered_ids( $connection_id, $post_ids ) {
 		global $wpdb;
 		$post_ids_sql = implode( ',', array_map( 'intval', $post_ids ) );
 
+		$order_join = $connection_id === Teams_Players_Relationship::NAME
```

Comment:
Just personal preference to surround comparisons with `()` for better readability.